x-pack/packages/kbn-elastic-assistant-common/impl/language_models/constants.ts

```diff
@@ -11,12 +11,17 @@ export const getDefaultArguments = (llmType?: string, temperature?: number, stop
         temperature: temperature ?? DEFAULT_BEDROCK_TEMPERATURE,
         stopSequences: stop ?? DEFAULT_BEDROCK_STOP_SEQUENCES,
       }
-    : { n: 1, stop: stop ?? null, temperature: temperature ?? DEFAULT_OPEN_AI_TEMPERATURE };
+    : (llmType === 'gemini' 
+      ? {
+          temperature: temperature ?? DEFAULT_GEMINI_TEMPERATURE,
```

Comment:
I think we should hold off on any gemini code to the assistant in this PR as it will not work until your next PR